lib/matplotlib/axes/_axes.py

```diff
@@ -2996,9 +3007,19 @@ def pie(self, x, explode=None, labels=None, colors=None,
             raise ValueError("Wedge sizes 'x' must be non negative values")
 
         sx = x.sum()
-        if sx > 1:
-            x = x / sx
 
+        if normalize is None:
+            if sx < 1:
+                cbook.warn_deprecated(
+                    "3.1", message="normalize=None gives the current behavior "
```

Comment:
```suggestion
                    "3.1", message="normalize=None does not normalize if the sum is less than 1 "
```

Be explicit about what "the current behavior" is.